Manuals for AKAI MPK Mini 2 and APC Mini (#152)
mixxxdj/manual: source/hardware/controllers/akai_mpk_mini_2.rst

> AKAI MPK Mini mkII

Reviewer: ```suggestion
Akai MPK Mini mkII
```

> Akai MPK Mini mkII

Reviewer: Can you rename the file to `akai_mpk_mini_mkii.rst`?

Author: sure

===================

> The *AKAI MPK Mini mkII* is a funny keyboard controller for musicians.

Reviewer: ```suggestion
The *Akai MPK Mini mkII* is a funny keyboard controller for musicians.
```

> This device is not really conceived for DJing but this adaptation can be funny.

Reviewer: ```suggestion
This device is not really conceived for DJing but could be useful for radio broadcasting setups. 
```

?

Author: Could be, I didn't think about radio broadcasting

> -  `Manufacturer’s product
>    page <https://www.akaipro.com/mpk-mini-mkii>`__

Reviewer: ```suggestion
-  `Manufacturer’s product page <https://www.akaipro.com/mpk-mini-mkii>`__
```

Mapping description
-------------------

> -  The main keyboard is mapped to the 64 Samplers slots for Skins that have them, since it has 25 keys, upper Samplers can be accessed by pressing the Octave Up/Down buttons.

Reviewer: ```suggestion
-  The main keyboard is mapped to the 64 Sampler slots for Skins that have them, since it has 25 keys. The upper Samplers can be accessed by pressing the Octave Up/Down buttons.
```

> -  The 8 pressure sentitive drum pads are attached to 6 Effects, 3 for FX1 and 3 for FX2, the last two pads are mapped for a fourth effect for when it will be available.

Reviewer: ```suggestion
-  The 8 pressure sensitive drum pads are attached to 6 Effects, 3 for FX1 and 3 for FX2, the last two pads are mapped for a fourth effect for when it will be available.
```

>    Tap gently on a pad to select it for editing, Tap hard to switch the effect On/Off.

Reviewer: ```suggestion
   Tap gently on a pad to select it for editing. Tap hard to switch the effect on/off.
```

   The red/green bank A/B currently does nothing.
-  The 8 knobs are attached to the controls of the selected effect, in order left-right, top-bottom.

Experiments with Samplers and Velocity

> -------------------

Reviewer: Please have a look at the Netlify deploy-preview:

```
12:32:21 AM: Warning, treated as error:
12:32:21 AM: /opt/build/repo/source/hardware/controllers/akai_mpk_mini_2.rst:20:Title underline too short.
12:32:21 AM: Experiments with Samplers and Velocity
12:32:21 AM: ------------------
```

Author: You're FAST... fixed it !

my local build didn't complain...
